### ISSUE TITLE
crypto-gcm.js: TypeError: Object #<Cipheriv> has no method 'getAuthTag'

### ISSUE THREAD

**Issue by rbarkas** [chris-rock/node-encryption-examples#1]
Running the unmodified sample code for crypto-gcm.js I get an error:

```
/home/sop/switch/test/cr.js:15
  var tag = cipher.getAuthTag();
               ^
TypeError: Object #<Cipheriv> has no method 'getAuthTag'
    at encrypt (/home/sop/switch/test/cr.js:15:20)
    at Object.<anonymous> (/home/sop/switch/test/cr.js:30:10)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by chris-rock** [comment]
@rbarkas which version of node are you using?

**Comment by rbarkas** [comment]
Hi Chris;

I just created a file using the demo code and ran it again and this time no
errors.

I can't explain it, perhaps a stray character during file creation the last
time when the error occurred.

In any event, I cannot now duplicate the error.

Thanks for replying, I'll let you know if it recurs.

I am using node 0.10.32

Roy Barkas

**Comment by chris-rock** [comment]
Are you sure you used node 0.10 and not 0.12?

**Comment by rbarkas** [comment]
Yes, only 0.10 installed.  I've been thinking about having a look at 12 but
haven't yet due to possible module compatibility issues

**Comment by chris-rock** [comment]
I am asking because the method `getAuthTag` is only available in node 0.11/0.12, but not in node 0.10. Therefore this sample should not work in node 0.10

**Comment by chris-rock** [comment]
please reopen if the issues persists
